mercadopago/px-ios
# (Added)[PXN-3660] - PXPaymentMethodSelector tracking

## What have changed?
We've added the capability to set up trackings for PXPaymentMethodSelector

## How to test:
Within the ExampleSwift app, use setTrackingConfiguration builder method, and check for headers on init checkout request. The flowID header, should be the same than the flowName passed through trackingConfiguration.

Also check for the events and screens tracking on PXTrackerListener extension for ViewController.

## Prints for layout changes:


## Extras:
